### ISSUE TITLE
Cancelled Tasks not deleting at top right

### ISSUE THREAD

**Issue by CMDR-Koda** [StrangeLoopGames/EcoIssues#19212]
Cancel a Task

It should have disappeared at the top right

Instead it piles up dead tasks along the top of the screen

No mods

**Comment by SlayksWood** [comment]
Can't reproduce, but we have a si,ilar bug #19112
@CMDR-Koda Thank you for the report. Can you provide a little more information.
1. Provide log files when it happens next time. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
2. Does it always happen or you have this a several times?
3. Can you describe what you were doing before the problem started?

**Comment by SlayksWood** [comment]
added for investigation. I also have complaints from several players. But noone have logs(

**Comment by ItsRosie** [comment]
Yeah, seems like a temperamental bug, I've not had it since I reported it, seems others have though...
